src/context.rs

```diff
@@ -131,16 +131,9 @@ impl ServiceContext {
         proto_infos: HashMap<ProtocolId, ProtocolInfo>,
         key_pair: Option<SecioKeyPair>,
         closed: Arc<AtomicBool>,
-        timeout: Duration,
```

Comment:
Why `timeout` is removed?

Reply:
This data is not used in context, so delete it